src/components/Submit/index.css

```diff
@@ -21,11 +26,10 @@
   }
 
   &:active {
-    box-shadow: none;
+    box-shadow: inset 0 0 0 1px color-mod(var(--green) shade(30%));
     position: relative;
     top: 0.25rem;
   }
-
   &:not(:first-child) {
```

Comment:
The linter is expecting an empty line before this rule.

Reply:
This should be resolved now